source/types.ts

```diff
@@ -37,6 +37,10 @@ export interface NextcloudClientInterface extends NextcloudClientProperties {
   checkConnectivity():                                                        Promise<boolean>;
   get(path: string):                                                          Promise<string | Buffer>;
```

Comment:
Why do you need to compile this ? Sounds like something is wrong

Reply:
Because I want to be able to use (for example) the type `FileDetails` or interface `OcsActivity` outside the module. As these are not compiled in the client (or helper), I cannot use these types.

Reply:
But that's a TS-only thing - the JS is provided for JS users. I'm fine with exposing `types.ts`, really. But why not create a `nextcloud-link.d.ts` instead of  adding yet another heavy build artifact?